docs/getting-started/Tachyons.stories.mdx

```diff
@@ -165,7 +165,7 @@ Here's a few common use cases to get you started.
 ```jsx
 <span className="flex items-center">
     Check
-    <CheckIcon className="ml1 fill-success-500" />
+    <CheckIcon className="ml1 fill-botanic-500" />
```

Comment:
botanic? is it an existing color in Sharegate specs? Otherwise I don't see how it is an improvement over "success" ?

Reply:
It is now, since success was role based, and could have been used for other reasons, we, me and Emie, decided that success was not a good color name, as it could also be used for a positive message. It is now on par with the sunray / marine naming convention.